frontend/src/components/atoms/Shared/PlusButton/PlusButton.tsx

```diff
@@ -0,0 +1,38 @@
+import {Link} from "react-router-dom";
+import {Fab} from "@material-ui/core";
+import {Add} from "@material-ui/icons";
+import {makeStyles} from "@material-ui/core/styles";
+
+type PlusButtonProps = {
+    readonly onLink: string;
+}
+
+const useStyles = makeStyles({
+        link: {
+            textDecoration: "none"
+        },
+        fab: {
+            padding: 10,
+            position: 'absolute',
+            right: 0,
+        },
+        icon: {
+            paddingRight: 7
+        },
+    }
+)
+
+const PlusButton = ({onLink}: PlusButtonProps) => {
+    const classes = useStyles();
+
+    return (
+        <Link to={onLink} className={classes.link}>
+            <Fab variant="extended" color="secondary" aria-label="add" className={classes.fab}>
+                <Add className={classes.icon}/>
+                Turniej
```

Comment:
To jest button na dodanie turnieju. Wiec jak dla mnie to nie atom i nie PlusButton tylko NewTournamentButton. Lepiej dzielic to featurowo niz tak technicznie :)